src/main/resources/application.properties

```diff
@@ -36,83 +26,36 @@ rabbitmq.queue.durable: true
-# Example 2: mongodb://username:mypasswd@localhost:27017                  - Single MongoDD instance with authentication
-# Example 3: mongodb://username:mypasswd@hostname1:27017,hostname2:27017  - Multiple MongoDB instances with authentication
-
-# To encrypt a password used in a uri the password must be given in it�s own property
-# Check configuration documentation for using jasypt encrypted passswords
-# Uri examples with encryption:
-# Example 1: 
-# spring.data.mongodb.password: ENC(en2h9ZQlZEuWyyfiTVrOPcQbRTwcXxOJ)                               - The encrypted password property
-# spring.data.mongodb.uri: mongodb://username:${spring.data.mongodb.password}@localhost:27017       - Single MongoDD instance with tag for password property
-
 spring.data.mongodb.uri: mongodb://localhost:27017
 spring.data.mongodb.database: eiffel_intelligence
-# Configure the session storage expiration time for logged in users of Eiffel Intelligence front-end
+
 server.session.timeout: 1200
-# Configure the database collection name to store sessions
 sessions.collection.name: sessions
 
-# Name of the database collection where aggregated objects are stored
 aggregations.collection.name: aggregations
-# Configure time-to-live value (seconds) for aggregated objects. This needs to be set if
-# using the example rule set 'AllEventsRules' to avoid the storage growing too large.
-# Recommended setting is 10 minutes.
 aggregations.collection.ttl:
-# The name of the database collection where to store the mapping between
-# aggregated objects and id of the Eiffel events that contributed to those
 event.object.map.collection.name: event_object_map
-# Name of the database collection where subscriptions are stored
 subscriptions.collection.name: subscriptions
-# Name of the database collection where we store pair of 'subscription:aggregated object'
-# where notification has been sent and should not be repeated
-subscriptions.repeat.handler.collection.name: subscriptions_repeat_handler
-# The waitlist contains Eiffel events which are waiting to be aggregated
-# Configure the database collection name for the waitlist
 waitlist.collection.name: wait_list
-# Set the time-to-live value for Eiffel events in the waitlist database (seconds)
 waitlist.collection.ttl: 600
-# Resend information for unprocessed events in waitlist, time in milliseconds
 waitlist.resend.initial.delay: 2000
 waitlist.resend.fixed.rate: 15000
-# Name of the collection where failed notifications are stored
 failed.notifications.collection.name: failed_notifications
-# Configure time-to-live for failed notifications in database (seconds)
 failed.notifications.collection.ttl: 600
-# Number of retries if a notification fail
 notification.retry: 3
 
-# Configure email sender and default subject for email notifications.
 email.sender: noreply@ericsson.com
 email.subject: Email Subscription Notification
 
-# Configure the email server to route email notifications through.
 spring.mail.host:
```

Comment:
Since we don't provide any example settings of this configuration should we link to Springs documentation? Or should we use the settings we configure to use for tests? Not here of course, but in the configuration.md file. If you think it's too much maybe we can add it as a separate issue to improve this part of the documentation?

Reply:
I couldn't find a solid place in the official docs where all this explained but there is a page in Baeldung that shows an example: https://www.baeldung.com/spring-email#2-spring-boot-mail-server-properties
Should we simply write an example of our own instead of linking to something external?

Reply:
I put an example configuration in the docs.